src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/CompilerServices/Symbols.vb

```diff
@@ -223,6 +223,7 @@ Namespace Microsoft.VisualBasic.CompilerServices
                 Case TypeCode.Char : Return GetType(Char)
                 Case TypeCode.String : Return GetType(String)
                 Case TypeCode.Object : Return GetType(Object)
+                Case TypeCode.DBNull : Return GetType(DBNull)
```

Comment:
The `GetTypeCode` extension hadn't been updated when `DBNull` was introduced to deal with the possibility of such values, which was a bug. Going through `Type.GetTypeCode` means that is now fixed, but this possible result has to be considered here. This seems to be the only place where the new possibility of `TypeCode.DBNull` has an impact.